lib/beanstream/payments_api.rb

```diff
@@ -129,9 +129,10 @@ def make_payment(payment)
       val = transaction_post("POST", make_payment_url, Beanstream.merchant_id, Beanstream.payments_api_key, payment)
     end
     
-    def complete_preauth(transaciton_id, amount)
+    def complete_preauth(transaciton_id, amount, order_number = nil)
       complete_url = make_payment_url+transaciton_id+"/completions"
-      completion = { :amount => amount }
+      completion = { amount: amount }
+      completion[order_number] = order_number unless order_number.nil?
```

Comment:
Shouldn't this be `completion[:order_number] = ...` (note the missing colon)
Another way to do this is like how you were doing it before, but with compact:
```
  completion = { amount: amount, order_number: order_number }.compact
```